src/frontend/utils/validators.ts

```diff
@@ -234,3 +233,13 @@ export const erBegrunnelseGyldig = (
         return ok(felt);
     }
 };
+
+export const erAdresseBeskyttet = (
+    adresseBeskyttelsesGradering: Adressebeskyttelsegradering | undefined
```

Comment:
Veldig nitpick men burde man tittalte å sende inn `null` her?

```suggestion
    adresseBeskyttelsesGradering: Adressebeskyttelsegradering | undefined | null
```